SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -52,42 +50,27 @@ TArray<FWorkerComponentData> EntityFactory::CreateEntityComponents(USpatialActor
 	Worker_EntityId EntityId = Channel->GetEntityId();
 
 	FString ClientWorkerAttribute = GetConnectionOwningWorkerId(Actor);
+	WorkerAttributeSet ServerWorkerAttributeSet = { SpatialConstants::DefaultServerWorkerType.ToString() };
```

Comment:
Similar to comments, above, I think we can refactor this to use the constants from SpatialConstants, since we no longer have multiple worker types.